Merge updated jaybee84/wordcount into draft-branch (#249)
jaybee84/ml-in-rd: content/17. ref-snippets.md

# Deriving disease modules from the compressed transcriptional space embedded in a deep autoencoder

[@doi:10.1038/s41467-020-14666-6]

Dwivedi, et. al. (2020) apply representation learning to meaningfully group genes into modules in a data-driven manner. 
The authors turned the deep neural network into a "glass-box" by layering knowledge-based interpretation on the gene modules identified using a variational autoencoder to provide insights into what each layer of their model represent.

# Pathway-level information extractor (PLIER) for gene expression data

[@doi:10.1038/s41592-019-0456-1]

Mao and colleagues (2019) use the principles of representation learning combined with prior-knowledge to develop a model that capitalizes on correlated gene expression to identify meaningful biological pathways and reduce noise in data introduced due to technical variation.
This method has been the foundation of mutiple recent works applying ML on rare disease gene expression data.

# Automatic detection of rare pathologies in fundus photographs using few-shot learning

[@doi:10.1016/j.media.2020.101660]

> Quellec, et. al. (2020) train convolutional neural network (CNN) is developed to identify rare pathologies in retinal images.

Reviewer: ```suggestion
Quellec, et. al. (2020) train a convolutional neural network (CNN) to identify rare pathologies in retinal images.
```
